src/kernel/kmain.zig

```diff
@@ -9,6 +9,7 @@ const vga = @import("vga.zig");
 const log = @import("log.zig");
 const serial = @import("serial.zig");
 const mem = @import("mem.zig");
+const options = @import("build_options");
```

Comment:
This is cool, how can you just pass in this string like that?

Reply:
I think that it's a hardcoded thing in the compiler and it just finds the right build options file :)